src/actions.cpp

```diff
@@ -78,7 +66,8 @@ bool Actions::registerEvent(Event_ptr event, const pugi::xml_node& node)
 		for (const auto& id : idList) {
 			auto result = useItemMap.emplace(id, std::move(*action));
 			if (!result.second) {
-				std::cout << "[Warning - Actions::registerEvent] Duplicate registered item with id: " << id << std::endl;
+				std::cout << "[Warning - Actions::registerEvent] Duplicate registered item with id: " << id
```

Comment:
We should replace these with `fmt::print`

Reply:
imo all error reporting should be moved to some function in order to:
- ensure the consistency of its format
- make all the changes in one place instead of 500 places like now

that would belong to a separate pull request though